source/solvers/multiphysics_interface.cc

```diff
@@ -1,16 +1,17 @@
 #include <solvers/multiphysics_interface.h>
 
+#include "solvers/free_surface.h"
 #include "solvers/heat_transfer.h"
 #include "solvers/tracer.h"
 
 
-
 template <int dim>
 MultiphysicsInterface<dim>::MultiphysicsInterface(
   const SimulationParameters<dim> &                            nsparam,
   std::shared_ptr<parallel::DistributedTriangulationBase<dim>> p_triangulation,
   std::shared_ptr<SimulationControl> p_simulation_control)
   : multiphysics_parameters(nsparam.multiphysics)
+  , verbosity(nsparam.linear_solver.verbosity)
```

Comment:
So are we sure we want to use the linear solver verbosity here? Maybe the non-linear solver verbosity would make more sense. Actually, maybe all we would need a single verbosity in the non-linear solver and if that verbosity is on, the iterative solver /linear solver verbosity would be on too.

Reply:
Yes you're right, I will change it for `nonlinear_solver` verbosity! I would not mind generalizing the verbosity to the linear solver, but are we sure that it is not necessary to keep them apart? Also it can be confusing for the users that a parameter in the nonlinear solver section changes something for the linear solver. 

I would advise, if we want a general verbosity parameter for the solvers, to pass it to a more general subsection of the prm file (like simulation control). From there, we can have: solver verbosity = quiet / verbose. And it would enable or disable verbosity for both. Would this suits what you had in mind?

Reply:
Yeah you are right. That would be the preferable route in the long term. Let's keep that in mind but keep it on the backburner for the moment. This PR has grown big enough as it is!